vm.go

```diff
@@ -327,17 +327,17 @@ func (vm *vm) setFuncName(s unistring.String) {
 
 func intToValue(i int64) Value {
 	if i >= -maxInt && i <= maxInt {
-		if i >= -128 && i <= 127 {
-			return intCache[i+128]
+		if i >= -8192 && i <= 8191 {
+			return intCache[i+8192]
 		}
 		return valueInt(i)
 	}
 	return valueFloat(i)
 }
 
 func int64ToValue(i int64) Value {
-	if i >= -128 && i <= 127 {
-		return int64Cache[i+128]
+	if i >= -8192 && i <= 8191 {
```

Comment:
[opt/q] This function didn't come up in the CPU profiles. Do you think it's worth increasing the cache size only for `intToValue`?

Reply:
I modified to keep these caches consistent but I don't have a strong opinion on it. I'll only modify intToValue.